db/migrate/20170824174144_create_user_alternative_uuids.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUserExternalUuids < ActiveRecord::Migration
```

Comment:
change file name to make class name

Reply:
@nathanstitt please change the filename to talk about external uuids, not alternative uuids (still reflects old model name)